pkg/domain/entities/network.go

```diff
@@ -41,19 +41,26 @@ type NetworkRmReport struct {
 
 // NetworkCreateOptions describes options to create a network
 type NetworkCreateOptions struct {
-	DisableDNS bool
-	Driver     string
-	Gateways   []net.IP
-	Internal   bool
-	Labels     map[string]string
-	MacVLAN    string
-	Ranges     []string
-	Subnets    []string
-	IPv6       bool
+	DisableDNS        bool
+	NetworkDNSServers []string
+	Driver            string
+	Gateways          []net.IP
+	Internal          bool
+	Labels            map[string]string
+	MacVLAN           string
+	Ranges            []string
+	Subnets           []string
+	IPv6              bool
 	// Mapping of driver options and values.
 	Options map[string]string
```

Comment:
@Luap99 Given this doesn't actually work with macvlan/ipvlan - should we make DNS servers an option?

Reply:
I think it would make sense but DnsEnabled is already a global option. And this comment is a bit to late given that we would need to change it in natavark and c/common again.

Reply:
Fair enough